### PR TITLE
Upgrade to Asciidoctor 1.5.4 (version 1.5.5-1)

### LINKED ISSUE #108: preview crashes with ifeval
I have a document like this

```
ifeval::["{backend}" == "pdf"]
== Note on PDF output
...
```

It causes the preview to not refresh/crash.

### PR DESCRIPTION
**Pull request by Mogztter**
- Bump version to 0.7.0
 - Remove unused dependencies
 - Fix dependency version

Remove `temp`, `opal-npm-wrapper` (transitive) and `wrench` dependency.

Resolves #108

### CODE REVIEW

**Comment by ldez** [comment]
@Mogztter can you rebase ?

File: lib/worker.coffee [hunk 1]
```diff
@@ -14,7 +14,7 @@ module.exports = (text, attributes, filePath) ->
 
   folder = path.dirname(filePath)
   Opal.ENV['$[]=']("PWD", path.dirname(attributes.opalPwd))
-  opts = Opal.hash2 ['base_dir', 'safe', 'doctype', 'attributes'],
+  opts = Opal.hash
```
Comment (Mogztter) [inline lib/worker.coffee]: @ldez I'm not sure about the syntax (coffee)...

**Comment by Mogztter** [comment]
@ldez Done!

File: lib/worker.coffee (hunk 1 above)
Comment (ldez) [inline lib/worker.coffee]: It's a object with attributes, you can remove `,` and `'`

**Comment by mojavelinux** [comment]
:+1:

**Pull request merged**